packages/shared/src/helpers/mapToGuidelineDesignTokens.ts

```diff
@@ -122,7 +122,10 @@ export const mapToGuidelineDesignTokens = (dataToTransform: Partial<Record<strin
 
     for (const [key, value] of Object.entries(enrichedDataToTransform)) {
         const designTokenName = TokenNameMapper[key] ?? key;
-        transformedDesignTokens[designTokenName] = transformDesignTokens(value) as TokenValues;
+        if (value) {
```

Comment:
Should we maybe explicitly check with:

```suggestion
        if (value !== null) {
```

Else we will skip all falsy values like empty strings, number 0 etc. Or is this never the case here?

Reply:
Good point, thanks! 👍🏻